src/Lecture1.hs

```diff
@@ -67,7 +71,11 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+
+lastDigit :: Int -> Int
+lastDigit x =   if x > 0
+                then  x `mod` 10
+                else  (x `rem` 10) * (-1)
```

Comment:
That's correct! 💯 
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```

Reply:
I was looking for something like that but couldn't find it, thanks!